src/action.js

```diff
@@ -1,7 +1,24 @@
+const fs = require("fs");
 const core = require("@actions/core");
 const exec = require("@actions/exec");
 const simpleGit = require("simple-git");
 
+/**
+ * Asynchronously reads the contents of a file.
+ * @param {string} filePath - The path of the file you want to read.
+ * @returns {Promise<string>} The contents of the file as a string.
+ */
+async function readFile(filePath) {
```

Comment:
maybe call this readUTF8File? Also, can you do this with [util.promisify](https://nodejs.org/api/util.html#util_util_promisify_original) instead of rolling your own?

Reply:
This is just copied from:
https://github.com/Codesee-io/codesee/blob/586e55aeb64458ff3706166602a32b4e31ea42f3/packages/codesee/src/init/util.js#L109-L123

Happy to adjust if you feel it's important.

Reply:
Not super important. We should clean it up at some point, though (less code === less bugs, right?)